docs/pages/api-reference/liveblocks-node.mdx

```diff
@@ -947,62 +947,473 @@ console.log(schema);
+  },
+});
+
+// { type: "comment", threadId: "th_d75sF3...", ... }
+console.log(editedComment);
+```
+
+A comment’s body is an array of paragraphs, each containing child nodes. Here’s
+an example of how to construct a comment’s body, which can be submitted under
+`data.body`.
+
+```tsx highlight="3-11,19"
+import { CommentBody } from "@liveblocks/node";
+
+const body: CommentBody = {
+  version: 1,
+  content: [
+    {
+      type: "paragraph",
+      children: [{ text: "Hello " }, { text: "world", bold: true }],
+    },
+  ],
+};
+
+const comment = await liveblocks.createComment({
+  roomId: "my-room-id",
+  threadId: "th_d75sF3...",
+
+  data: {
+    // The comment's body, uses the `CommentBody` type
+    body,
+
+    // ...
+  },
+});
+```
+
+```ts
+const editedComment = await liveblocks.editComment({
+  roomId: "my-room-id",
+  threadId: "th_d75sF3...",
+  commentId: "cm_agH76a...",
+
+  data: {
+    // The comment's body, uses the `CommentBody` type
+    body: {
+      version: 1,
+      content: [
+        /* The comment's body text goes here, see above */
+      ],
+    },
+
+    // The ID of the user that edited the comment
+    userId: "alicia@example.com",
+
+    // Optional, the time the comment was edited
+    editedAt: new Date(),
+  },
+});
+
+// { type: "comment", threadId: "th_d75sF3...", ... }
+console.log(editedComment);
+```
+
+#### Liveblocks.deleteComment [#post-rooms-roomId-threads-threadId-comments-commentId] [@badge=Beta]
+
+Deletes a specific comment from a thread within a room. If there are no
+remaining comments in the thread, the thread is also deleted. This method throws
+an error if the comment isn’t found. This is a wrapper around the
+[Delete Comment API](/docs/api-reference/rest-api-endpoints#post-rooms-roomId-threads-threadId-comments-commentId)
+and returns no response.
+
+```ts
+await liveblocks.deleteComment({
+  roomId: "my-room-id",
+  threadId: "th_d75sF3...",
+  commentId: "cm_agH76a...",
+});
+```
+
+#### Liveblocks.addCommentReaction [#post-rooms-roomId-threads-threadId-comments-commentId-add-reaction] [@badge=Beta]
+
+Adds a reaction to a specific comment in a thread within a room. Throws an error
+if the comment isn’t found or if the user has already added the same reaction on
+the comment. This is a wrapper around the
+[Add Comment Reaction API](/docs/api-reference/rest-api-endpoints#post-rooms-roomId-threads-threadId-comments-commentId-add-reaction)
+and returns the new reaction.
+
+```ts
+const reaction = await liveblocks.addCommentReaction({
+  roomId: "my-room-id",
+  threadId: "th_d75sF3...",
+  commentId: "cm_agH76a...",
+
+  data: {
+    emoji: "👨‍👩‍👧",
+    userId: "guillaume@example.com",
+    createdAt: new Date(), // Optional, the time the reaction was added
+  },
+});
+
+// { emoji: "👨‍👩‍👧", userId "guillaume@example.com", ... }
+console.log(reaction);
+```
+
+#### Liveblocks.deleteCommentReaction [#post-rooms-roomId-threads-threadId-comments-commentId-remove-reaction] [@badge=Beta]
```

Comment:
```suggestion
#### Liveblocks.removeCommentReaction [#post-rooms-roomId-threads-threadId-comments-commentId-remove-reaction] [@badge=Beta]
```

Reply:
Yes, renaming this to `removeCommentReaction`